### DISCUSSION TITLE
Curious why await request.get_json() returns None with chunked transfer encoding but  await request.data works?

### DISCUSSION THREAD

**Discussion by space88man** [pallets/quart#360] [Q&A]
Situation: client POST'ing using chunked transfer encoding. with either uvicorn/hypercorn frontend with a Quart ASGI app. 

When I use `await request.get_json()` it always returns `None` but `await request.data` will work and
get the full payload. I can't control the client ATM.

Is this the expected behaviour?

I'm actually ok with  `json.loads(...request.data...)` but curious about the difference in behaviour - if it matters the payload is normally very small and completed in the first chunk.

**Comment by catwell** [comment]
The most likely possibility to me is that the client does not set the JSON mimetype. What happens if you do `request.get_json(force=true)`?
